app/experimenter/experiments/api/v4/views.py

```diff
@@ -7,10 +7,10 @@
 class ExperimentRapidViewSet(
     mixins.RetrieveModelMixin, mixins.ListModelMixin, viewsets.GenericViewSet,
 ):
-    lookup_field = "slug"
+    lookup_field = "normandy_slug"
     queryset = (
         Experiment.objects.get_prefetched()
         .filter(type=Experiment.TYPE_RAPID)
-        .exclude(status=Experiment.STATUS_DRAFT)
+        .exclude(status__in=[Experiment.STATUS_DRAFT, Experiment.STATUS_REVIEW])
```

Comment:
wait what's this for

Reply:
oh cuz it doesn't hav ea normandy slug

Reply:
@tiftran Don't you have to update this? https://github.com/mozilla/experimenter/blob/bee394a241175880dc8611a506871e4b5e0f0caa/app/experimenter/experiments/tests/api/v4/test_views.py#L25-L26

Reply:
ooh right right

Reply:
Oh! the tests pass because we're using the ExperimentFactory that just always generates a normandy slug!  D'oh!  God we need that rapid factory XD

Reply:
we do? maybe I'm missing something but seems like https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/tests/factories.py#L158 is the only line that has a normandy slug generated, but we're calling create_with_variants, instead of create_with_status? was assuming that since the normandy slug is None, we're comparing sets of None?

Reply:
OH! XD the test was never valid!  Goooooood eye @tiftran thnx for fixing ;)